pkg/shed/db.go

```diff
@@ -81,6 +84,7 @@ func NewDB(path string, metricsPrefix string) (db *DB, err error) {
 func (db *DB) Put(key []byte, value []byte) (err error) {
 	err = db.ldb.Put(key, value, nil)
 	if err != nil {
+		db.logger.Debugf("failed to insert in to DB. Error : ",  err.Error())
```

Comment:
This is still not functionally right. Missing `%s`.

Reply:
oops.. sorry..